public-ui/src/app/data-browser/views/fmh-view/fmh-view.component.ts

```diff
@@ -69,8 +70,36 @@ export class FmhViewComponent implements OnInit {
       },
       complete: () => { this.fmQuestionFetchComplete = true; }
     }));
+    this.subscriptions.push(this.searchText.valueChanges
+      .debounceTime(1500)
+      .distinctUntilChanged()
+      .switchMap((query) => this.api.getFMHGroupedQuestions('43528698',
+        query, conditionQuestionConceptIds))
+      .subscribe({
+        next: results => {
+          this.processQuestions(results, 'condition');
+        },
+        error: err => {
+          console.log('Error searching: ', err);
+          this.loading = false;
+        }
+      }));
+    this.subscriptions.push(this.searchText.valueChanges
+      .debounceTime(1500)
+      .distinctUntilChanged()
+      .switchMap((query) => this.api.getFMHGroupedQuestions('43528698',
+        this.searchText.value, fmQuestionConceptIds))
+      .subscribe({
+        next: results => {
+          this.processQuestions(results, 'family_member');
+        },
+        error: err => {
+          console.log('Error searching: ', err);
+          this.loading = false;
+        }
+      }));
   }
-
```

Comment:
Each time search keyword changes fmh question result changes too. So send the matching question count from fmh component to parent component survey-view.